releasenotes.md

```diff
@@ -12,6 +12,7 @@ Released on xx/xx/xxxx.
 - Allow simulations and forecast to work across the end of the year to the next year. This is for [#239](https://github.com/ibpsa/project1-boptest/issues/239).
 - Pin base Docker image to ``linux/x86_64`` platform. This is for [#608](https://github.com/ibpsa/project1-boptest/issues/608).
 - Correct typo in design documentation about connecting inputs to overwrite blocks in wrapper model. This is for [#601](https://github.com/ibpsa/project1-boptest/issues/601).
+- Correct Git LFS configuration in the ``testing/Dockerfile`` image used in tests and compilation. This should allow Spawn of EnergyPlus emulators to be compiled with the default compilation pipeline. This is for [#613](https://github.com/ibpsa/project1-boptest/issues/613).
```

Comment:
Happy to remove this if it makes you feel more comfortable about merging this change as well:
```suggestion
- Correct Git LFS configuration in the ``testing/Dockerfile`` image used in tests and compilation. This is for [#613](https://github.com/ibpsa/project1-boptest/issues/613).
```